.github/actions/setup-env/action.yml

```diff
@@ -35,7 +35,7 @@ inputs:
     type: string
     description: 'The version of Java to install'
     required: false
-    default: '11'
+    default: '17'
```

Comment:
Should probably update https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/.github/actions/setup-java-env/action.yml#L33
as well